jovian/jovian_nb_ext/main.js

```diff
@@ -520,12 +535,13 @@ define([
             .text("False")
         );
       const artifacts_label = $("<label/>").text(
-        "Any outputs files or artifacts generated from the modeling processing. This can include model weights/checkpoints, generated CSVs, images etc. - Pass the list of strings(filenames)"
+        "Any outputs files or artifacts generated from the modeling processing. This can include model weights/checkpoints, generated CSVs, images etc. - Pass the list of strings(filenames) such as `jvn.png, README.md`"
```

Comment:
Examples can be `submission.csv, weights.h5`

Reply:
I will change :)